_episodes_rmd/05-dplyr.Rmd

```diff
@@ -57,14 +51,14 @@ packages give you access to more functions. You need to install a package and
 then load it to be able to use it.
 
 ```{r, eval = FALSE, purl = FALSE}
-install.packages("dplyr") ## install
+install.packages("tidyverse") ## install
 ```
 
 You might get asked to choose a CRAN mirror -- this is asking you to
 choose a site to download the package from. The choice doesn't matter too much; I'd recommend choosing the RStudio mirror.
 
 ```{r, message = FALSE, purl = FALSE}
-library("dplyr")          ## load
+library("tidyverse")          ## load
```

Comment:
Same here

Reply:
addressed in new commit of this pull request